releng/dockerfiles/Dockerfile

```diff
@@ -17,6 +17,7 @@ WORKDIR /app/
 
 RUN mvn clean install -DskipTests -Dskip-rcp -Pfilter-grammar -Pctf-grammar
 
-FROM alpine:3.16.0
+FROM alpine:3.18.0
+
+COPY --from=packager /root/.m2/ /root/.m2/
```

Comment:
Just curious, why is the whole .m2 directory copied now instead of the just the tracecompass sub-directory. Please let me know.

Reply:
The whole directory is copied because some files outside of that are needed to build the incubator and the trace-server.